### PR TITLE
Meet community standards

### PR DESCRIPTION
**Pull request by wordshaker**
## Description Of Change

Make change to the repository to ensure it meets the community standards outlined on the GitHub Insights tab

image

## Outline of changes

- Add Code of Conduct file that links to [the .NET Foundation Code of Conduct](https://dotnetfoundation.org/about/code-of-conduct)
- Add Contribution.md
- Update README information and add pretty images
- Make language consistent across files to match the README. Change `working group` to `committee` (can change the other way if preferred)
- Add issue template for repo improvements
- Update other issue templates

## Purpose Of Change

To make us closer align to the suggested baselines for community projects

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+- Fill in the details and submit
+
+# Raising Pull Requests
+
+- Fork the repository
+- Open a branch and commit your changes
+- Raise a pull request and fill in the information asked for in the description
+- The membership committee will automatically be added as reviewers as code owners of this repository
+
+# Licensing
```
Comment (wordshaker) [inline .github/CONTRIBUTING.md:51]: I don't actually know what licence we have on this repo / .NET Foundation work. @BillWagner do you know?

**Review by jguadagno — APPROVED** [review]
Thanks @wordshaker!

File: .github/CONTRIBUTING.md (hunk 1 above)
Comment (BillWagner) [reply .github/CONTRIBUTING.md:51]: The latest Creative Commons would be the right choice: This is a text / content repo.

**Review by BillWagner — APPROVED** [review]
This is great @wordshaker 

Thank you!

I'll :shipit: now.

**Pull request merged**